### PR TITLE
Add tabs navigation

### PR DESCRIPTION
**Pull request by Almond-Peach**
[As a user I should be able to navigate between the tabs](https://trello.com/c/A6MUCGoW)

### CODE REVIEW

**Review by ManukyanQristine — APPROVED** [review]

File: app/src/main/java/com/simply/birthdayapp/presentation/navigation/NavBar.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.navigation.NavController
+import androidx.navigation.compose.currentBackStackEntryAsState
+
+@Composable
+fun NavBar(
+    items: List<NavBarItem>,
+    navController: NavController,
+    onItemClick: (NavBarItem) -> Unit,
```
Comment (artur-ghonyan) [inline app/src/main/java/com/simply/birthdayapp/presentation/navigation/NavBar.kt]: Add default value

File: app/src/main/java/com/simply/birthdayapp/presentation/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import androidx.activity.compose.setContent
+import com.simply.birthdayapp.presentation.navigation.TopLevelNavigation
+import com.simply.birthdayapp.presentation.ui.theme.BirthdayAppTheme
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            BirthdayAppTheme {
+                TopLevelNavigation()
```
Comment (artur-ghonyan) [inline app/src/main/java/com/simply/birthdayapp/presentation/MainActivity.kt]: Rename this function.

File: app/src/main/java/com/simply/birthdayapp/presentation/navigation/NavBar.kt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                        contentDescription = it.name,
+                        tint = if (selected) Color.Black else Color.Unspecified
+                    )
+                },
+            )
+        }
+    }
+}
+
+data class NavBarItem(
```
Comment (artur-ghonyan) [inline app/src/main/java/com/simply/birthdayapp/presentation/navigation/NavBar.kt]: Move to separate file

File: app/src/main/java/com/simply/birthdayapp/presentation/navigation/TopLevelNavigation.kt [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        startDestination = TopLevelDestination.Tabs.route,
+    ) {
+        composable(TopLevelDestination.Tabs.route) {
+            TabsScreen()
+        }
+    }
+}
+
+sealed class TopLevelDestination(val route: String) {
+    object Tabs : TopLevelDestination("tabs")
```
Comment (artur-ghonyan) [inline app/src/main/java/com/simply/birthdayapp/presentation/navigation/TopLevelNavigation.kt]: Change object name and key

**Review by artur-ghonyan — CHANGES_REQUESTED** [review]

**Review by ManukyanQristine — APPROVED** [review]

**Review by artur-ghonyan — APPROVED** [review]

**Pull request merged**